swagger.yaml

```diff
@@ -2342,8 +2342,13 @@ definitions:
     properties:
       next_instruction_seconds:
         type: integer
-      exit_on_completion:
-        type: boolean
+      post_step_action:
```

Comment:
@avishayt do you have a suggestion for a name? not sure how clear it is.
It represent what will happen in the next iteration.

Reply:
It's been approved by @avishayt. I think `post_step_action` reflects well what action will happen after a step. Continue will continue to handle steps, exit will stop handling steps and exit.